docs/getting-started/11-expose-function.md

```diff
@@ -0,0 +1,194 @@
+
+> **NOTE:** Learn also how to [secure the Function](/components/api-gateway#tutorials-expose-and-secure-a-service-deploy-expose-and-secure-the-sample-resources).
+
+## Steps
+
+### Expose the Function
+
+Follow these steps:
+
+<div tabs name="steps" group="expose-function">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Create an APIRule CR for the Function. It is exposed on port `80`, which is the default port of the [Service](/components/serverless/#architecture-architecture).
+
+  ```yaml
+  cat <<EOF | kubectl apply -f -
+    apiVersion: gateway.kyma-project.io/v1alpha1
+    kind: APIRule
+    metadata:
+      name: orders-function
+      namespace: orders-service
+    spec:
+      gateway: kyma-gateway.kyma-system.svc.cluster.local
+      rules:
+        - path: /.*
+          accessStrategies:
+            - config: {}
+              handler: noop
+          methods: ["GET","POST"]
+      service:
+        host: orders-function
+        name: orders-function
+        port: 80
+  EOF
+  ```
+
+2. Check that the APIRule was created and has the `OK` status:
+
+  ```bash
+  kubectl get apirules orders-function -n orders-service -o=jsonpath='{.status.APIRuleStatus.code}'
+  ```
+
+3. Access the Function's external address:
+
+   ```bash
+   curl https://orders-function.{CLUSTER_DOMAIN}
+   ```
+
+    </details>
+    <details>
+    <summary label="console-ui">
+    Console UI
+    </summary>
+
+1. From the drop-down list in the top navigation panel, select the `orders-service` Namespace.
+
+2. Go to **Configuration** > **API Rules** in the left navigation panel and select **Create API Rule**.
+
+3. In the **General settings** section:
+
+    - Enter `orders-function` as the API Rule's **Name**.
+
+    >**NOTE:** The APIRule CR can have a name different from that of the Function, but it is recommended that all related resources share common names.
+
+    - Enter `orders-function` as **Hostname** to indicate the host on which you want to expose your Function.
+
+    - Select `orders-function` as the **Service** that indicates the Function you want to expose.
+
+4. In the **Access strategies** section, leave the default settings, with the `GET` and `POST` methods and the `noop` handler selected.
+
+5. Select **Create** to confirm the changes.
+
+    A message appears on the screen confirming that the changes were saved.
+
+6. Once the pop-up box closes, check that you can access the Function by selecting the HTTPS link under the **Host** column of the new `orders-function` API Rule.
+
+    </details>
+</div>
+
+### Call and test the microservice
+
+Follow these steps:
+
+> **CAUTION:** If you have a Minikube cluster, you must first add the IP address of the exposed Service to the `hosts` file on your machine:
+>
+> ```bash
+> echo "$(minikube ip) orders-function.kyma.local" | sudo tee -a /etc/hosts
+> ```
+
+1. Retrieve the domain of an exposed Function and save it to an environment variable:
```

Comment:
```suggestion
1. Retrieve the domain of the exposed Function and save it to an environment variable:
```
I believe we should stick to the definite article (or the possessive adjective `your`) here, as this is the guide on Exposing Function, and you open it with the words: 
```
In this guide, you will expose your Function outside the cluster, through an HTTP proxy. 
```
where you already mention the Function. 
Plus you also use the definite article in the steps for kubectl above.